mix.exs

```diff
@@ -64,10 +64,15 @@ defmodule JSONAPI.Mixfile do
         "Mitchell Henke",
         "Jake Robers",
         "Sean Callan",
-        "James Herdman"
+        "James Herdman",
+        "Mathew Polzin"
```

Comment:
Added @mattpolzin as a maintainer since he will be the primary driver for this lib 🎉